docs/usage.md

```diff
@@ -8,7 +8,7 @@ fetch-release > nix-bitcoin-release.nix
 
 Nodeinfo
 ---
-Run `nodeinfo` to see your onion addresses for the webindex, spark, etc. if they are enabled.
+Run `nodeinfo` to see onion addresses and local addresses for enabled services.
```

Comment:
There are two other lines that need to be changed in usage.md (`grep '{SSHD_ONION,ELECTRS_ONION}'` won't work)

Reply:
Fixed in `fixup! update usage.md`.